products/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from .views      import ProductView
+
+urlpatterns = [
+    path('/<int:product_group_id>', ProductView.as_view()),
```

Comment:
```suggestion
    path('/group/<int:product_group_id>', ProductView.as_view()),
```
products라는 url뒤에 바로 숫자가 오면, RESTful URL규칙상 상세 상품의 데이터가 됩니다 :)
따라서 앞에 `group`이 있는 것이 보다 이해하기 쉽겠어요!

Reply:
소헌님 이게 상세 상품의 데이터가 맞아요!
마켓컬리는 `product group` 안에 상세 상품을 선택하게 되어있는거라 `product group` 전체의 `description` 은 공유하고 상세 상품 페이지가 따로 존재하지 않아서 결과적으로는 `group_id` 로 들어가는 페이지 = 상세상품 페이지 라고 판단하여 요런 식으로 url 을 넣었습니다😁...! 바꾸는게 좀 더 명확할까용?

Reply:
수아님! **데이터 기준**으로 생각해보면, 명확히는 `product group 리스트`에서 `하나의 product group`을 선택하면 상세 페이지를 반환해주죠. 정확히는 상품이 아니라 `상품 그룹`입니다. 그렇기 때문에 URI는 `product-group/<int:product_group_id>`가 되야 합니다.